routing/router_test.go

```diff
@@ -110,6 +112,7 @@ func createTestCtxFromGraphInstance(startingHeight uint32, graphInstance *testGr
 		router:    router,
 		graph:     graphInstance.graph,
 		aliases:   graphInstance.aliasMap,
+		privKeys:  graphInstance.privKeyMap,
```

Comment:
This looks like it belongs more to the previous commit where priv keys are introduced.

Reply:
Fixed

Reply:
Done